src/extra/components/_common.py

```diff
@@ -34,10 +34,6 @@ def _instrument_to_sase(instrument):
         return 3
 
 
-def _identify_sase(data):
-    return _instrument_to_sase(_identify_instrument(data))
-
-
 def _select_subcomponent_trains(src, keys, dst=None):
```

Comment:
Did you mean to get rid of the rest of this module & move it into `.utils`?

Reply:
Not quite, I meant to merge anything that is **used** into `.utils`. That particular function `extra.components._common._identify_sase` was actually never used, but there's an `extra.components.utils.identify_sase` that is. It is meant to keep the latter and drop the former.

Reply:
That makes sense, but I'm wondering why you haven't removed this file entirely?

Reply:
Because I didn't `git add` the removal...

Reply:
Aha, thank-you!